WordPress/gutenberg
# Patterns: removes the pattern experiment

## What?
Removes the Patterns experiment added in #50533

## Why?
This is not likely to be used now as Patterns are instead being merged with reusable blocks

## How?
Takes out what was added in

## Testing Instructions

- With TT3 theme in editor code view add ``
- Switch to editor view and make sure the pattern is added as expected and displays in frontend

https://github.com/WordPress/gutenberg/assets/3629020/d4320e8a-506f-4371-b88f-aaf51f04b78b
